iml-wasm-components/iml-grafana-chart/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use seed::{iframe, attrs, prelude::*};
+
+/// Create an iframe that loads the specified stratagem chart
+pub fn grafana_chart<T>(dashboard_id: &str, dashboard_name: &str, refresh: &str, panel_id: u16, width: &str, height: &str) -> El<T> {
+    iframe![
+        attrs! { 
+            At::Src => format!("https://localhost:8443/grafana/d-solo/{}/{}?orgId=1&refresh={}&panelId={}", dashboard_id, dashboard_name, refresh, panel_id),
```

Comment:
We can't rely on this url being constant.

Use `iml_environment::ui_root()` which provides the current base_uri

Reply:
Will that return 
```
https://localhost:8443/
```
?

I looked at the source and it looks like it's doing 
```
document().base_uri().unwrap().unwrap_or_default()
```
Calling base_uri will return the entire url won't it?

Reply:
Depends really. For example:

![Screen Shot 2019-07-12 at 2 46 24 PM](https://user-images.githubusercontent.com/458717/61151422-ea637a80-a4b3-11e9-982c-4c6be1d602d0.png)


The reason for this is due to this line:

https://github.com/whamcloud/GUI/blob/75e184be495a808b3bbaf4fc982cc8b5401151b2/index.ejs#L8

You were most likely testing within the old gui iframe which probably doesn't have that tag.

Reply:
Since this component will be embedded in the GUI, I don't think there is an issue here.

Reply:
I.E. you should be able to use `ui_root()`.

Reply:
I will just need to replace `ui` with `grafana` correct?

Reply:
Yeah... may as well create a `grafana_root()` fn within `iml_environment` which internally calls `ui_root` and does that.

Reply:
Sounds good 👍